test/changelog/oban_workers/stats_processor_test.exs

```diff
@@ -1,24 +1,42 @@
-defmodule ChangelogStatsTest do
+defmodule Changelog.ObanWorkers.StatsProcessorTest do
   use Changelog.SchemaCase
+  use Oban.Testing, repo: Changelog.Repo
```

Comment:
If there were more Oban tests, this could go right in `Changelog.SchemaCase`. However, since this is the only module with Oban-specific tests, I only stuck it in here.